crates/devolutions-pedm/src/api/state.rs

```diff
@@ -65,11 +60,11 @@ where
     }
 }
 
-impl FromRef<AppState> for Arc<RwLock<Policy>> {
-    fn from_ref(state: &AppState) -> Self {
-        Arc::clone(&state.policy)
-    }
-}
+// impl FromRef<AppState> for Policy {
+//     fn from_ref(state: &AppState) -> Self {
+//         &state.policy
+//     }
+// }
```

Comment:
Feel free to delete if it’s not useful, it’s not very tricky code either

Reply:
Refactoring leftovers, gone